model/task.js

```diff
@@ -163,14 +163,19 @@ module.exports = function(app, callback) {
 					}
 					var pa11yOptions = {
 						standard: task.standard,
+						includeWarnings: true,
+						includeNotices: true,
 						timeout: (task.timeout || 30000),
 						wait: (task.wait || 0),
 						ignore: task.ignore,
 						actions: task.actions || [],
-						phantom: {},
+						chromeLaunchConfig: {
+							args: ['--no-sandbox']
```

Comment:
`--no-sandbox` shouldn't be the default option. The Chromium team specifically warn against using it, so the fact that we're using it in the first place to get it running is the wrong thing to do 😆 so it'd be doubly-triply wrong to force it on all users, some of whom may actually have more appropriate kernel setups.